apps/codeprop/Makefile.codeprop-tmp

```diff
@@ -0,0 +1,9 @@
+codeprop-tmp_src = codeprop-tmp.c
+
+# Enable large memory model supports for WISMOTE and EXP5438 platform
```

Comment:
@Simonduq
I have done few other changes as per your proposal. Please look into it.